README.md

```diff
@@ -22,113 +22,113 @@ These numbers are derived from local simulations with fixed inputs (see [Run ben
 <details>
 <summary><b>Run options</b></summary>
 
-Last run: Wed, 21 Feb 2024 05:53:32 GMT
-| Option              |   Value |
-| :------------------ | ------: |
-| L2 gas price (Gwei) | `0.004` |
-| L1 gas price (Gwei) |    `40` |
-| ETH price (USD)     | `$2500` |
+Last run: Thu, 25 Apr 2024 18:45:57 GMT
+| Option                         |         Value |
```

Comment:
Should we include a README section that provides links to data sources, such as https://etherscan.io/gastracker, to reference the numbers? This would allow us to consistently update the figures from the same source in the future.

Reply:
Would look to a dune dashboard

Reply:
Yep a dune dashboard would be great